src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -79,6 +79,10 @@ public class EventHubAdapterFactory : IQueueAdapterFactory, IQueueAdapter, IQueu
         /// Create a queue mapper to map EventHub partitions to queues
         /// </summary>
         protected Func<string[], IEventHubQueueMapper> QueueMapperFactory { get; set; }
+        /// <summary>
+        /// Create a receiver monitor to report performance metrics.
+        /// </summary>
+        protected Func<string, string, Logger, IEventHubReceiverMonitor> ReceiverMonitorFactory { get; set; }
```

Comment:
would be useful to explain the type parameters for the Func in the XML document, since the signature without argument names is not self explanatory.